packages/stream_chat_flutter/lib/src/message_widget/parse_attachments.dart

```diff
@@ -102,8 +102,11 @@ class ParseAttachments extends StatelessWidget {
       }
     };
 
+    final userId = StreamChat.of(context).currentUser?.id;
+
     // Create a default attachmentBuilders list if not provided.
     final builders = StreamAttachmentWidgetBuilder.defaultBuilders(
+      userId: userId,
```

Comment:
No need to pass as I have commented in file 1.

Reply:
We are not able to access context in function, so passed from here.